modules/core_project_factory/main.tf

```diff
@@ -104,7 +103,7 @@ resource "google_resource_manager_lien" "lien" {
 resource "google_project_service" "project_services" {
   count = "${length(var.activate_apis)}"
 
-  project = "${local.project_id}"
+  project = "${google_project.main.project_id}"
```

Comment:
We should keep `local.project_id` unless there's a reason we can't, similarly for all cases of this replacement.

Reply:
I am of the opinion that the project_id local should be removed or repurposed to make the project_id generation more clear. It also adds a layer of redirection that seems unnecessary so my comments on dropping the `project_number` local applies as well.